content/ko/docs/concepts/security/multi-tenancy.md

```diff
@@ -217,8 +217,10 @@ _소란스러운 이웃_ 문제를 예방하기 위한 목적을 가지고 있
 의해 제어될 수 있다. 테넌트 간의 엄격한 네트워크 격리가 필요한 멀티 테넌트 환경에서는, 
 파드 간의 통신을 거부하는 기본 정책과 모든 파드가 네임 해석(name resolution)을 위해 
 DNS 서버를 쿼리하도록 하는 규칙을 시작에 설정하는 것을 권고한다. 이러한 기본 정책을 
-기반으로 하여, 네임스페이스 내에서 통신을 허용하는 관대한 규칙을 추가할 수 있다. 이러한 
-방식은 요구에 따라 한 단계 더 정제할 수 있다. 이는 하나의 컨트롤 플레인 내의 파드에 
+기반으로 하여, 네임스페이스 내에서 통신을 허용하는 관대한 규칙을 추가할 수 있다.
+또한 네임스페이스 간에 트래픽을 허용해야 하는 경우 네트워크 정책 정의의 
+namespaceSelector 필드에 빈 레이블 셀렉터 '{}'를 사용하지 않는 것이 좋다.

```

Comment:
> It is also recommended not to use empty label selector '{}' for namespaceSelector field in network policy definition, in case traffic need to be allowed between namespaces.

namespaceSelector라고 표기하면 될까요? [91ecbb9](https://github.com/kubernetes/website/commit/91ecbb977cf0dd7a8632f0a35c186689a1a486ec)에 대한 변경사항입니다!

Reply:
[한글화 가이드 방침](https://kubernetes.io/ko/docs/contribute/localization_ko/#api-%EC%98%A4%EB%B8%8C%EC%A0%9D%ED%8A%B8-%EC%9A%A9%EC%96%B4-%ED%95%9C%EA%B8%80%ED%99%94-%EB%B0%A9%EC%B9%A8) 에 따르면,
> `kind`에 속하지 않는 API 오브젝트는 원문 그대로 표기한다

고 되어 있어서, 저는 지금처럼 영문 그대로 표기해도 될 것 같다고 생각합니다 !!

Reply:
[네임스페이스 셀렉터](https://kubernetes.io/ko/docs/concepts/scheduling-eviction/assign-pod-node/#namespace-selector)